website/static/js/pages/project-settings-page.js

```diff
@@ -67,14 +67,16 @@ $(document).ready(function() {
     }
     var disableCategory = !window.contextVars.node.parentExists;
     // need check because node category doesn't exist for registrations
-    if ($('#nodeCategorySettings').length) {
-        var categorySettingsVM = new ProjectSettings.NodeCategorySettings(
-            window.contextVars.node.category,
-            categories,
-            window.contextVars.node.urls.update,
-            disableCategory
-        );
-        $osf.applyBindings(categorySettingsVM, $('#nodeCategorySettings')[0]);
+    if ($('#projectSettings').length) {
+        var projectSettingsVM = new ProjectSettings.ProjectSettings( {
+            currentTitle: ctx.node.title,
+            currentDescription: ctx.node.description,
+            category: ctx.node.category,
+            categories: categories,
```

Comment:
I'm a bit swamped to dig there myself- but one other place you might look is the mako template that uses the viewmodel. It may also reference some of these attributes.